test/watt_config.yml

```diff
@@ -0,0 +1,49 @@
+aggregate_prs_results_test:
+  workflow_name: "AggregatePRSResults"
+  test_name: "Base"
+  path: "/ImputationPipeline/AggregatePRSResults.wdl"
+  test_inputs: "/test/AggregatePRSResults/test_inputs.json"
+  expected_outputs: "/test/AggregatePRSResults/test_outputs.json"
+
+benchmark_vcfs_ignore_filters_test:
+  workflow_name: "BenchmarkVCFs"
+  test_name: "IgnoreFilters"
+  path: "/BenchmarkVCFs/BenchmarkVCFs.wdl"
+  test_inputs: "/test/BenchmarkVCFs/test_ignore_filters_input.json"
+  expected_outputs: "/test/BenchmarkVCFs/test_ignore_filters_outputs.json"
+
+benchmark_vcfs_test:
+  workflow_name: "BenchmarkVCFs"
+  test_name: "Base"
+  path: "/BenchmarkVCFs/BenchmarkVCFs.wdl"
+  test_inputs: "/test/BenchmarkVCFs/test_base_inputs.json"
+  expected_outputs: "/test/BenchmarkVCFs/test_base_outputs.json"
+
+benchmark_vcfs_unmatching_genotypes_test:
+  workflow_name: "BenchmarkVCFs"
+  test_name: "UnmatchingGenotypes"
+  path: "/BenchmarkVCFs/BenchmarkVCFs.wdl"
+  test_inputs: "/test/BenchmarkVCFs/test_unmatching_genotypes_inputs.json"
+  expected_outputs: "/test/BenchmarkVCFs/test_unmatching_genotypes_outputs.json"
+
+compare_benchmarks_test:
+  workflow_name: "CompareBenchmarks"
+  test_name: "Base"
+  path: "/BenchmarkVCFs/CompareBenchmarks.wdl"
+  test_inputs: "/test/CompareBenchmarks/test_base_inputs.json"
+  expected_outputs: "/test/CompareBenchmarks/test_base_outputs.json"
+
+
+## proposed new format
```

Comment:
This is not a bad idea. I'd be up for changing Watt to do something like this. I'm not a fan of having the empty `-` lines, so maybe something like:

```
workflow_name:
  path: "/my/path"
  tests:
    test_name:
      test_inputs: "/here/they/are"
      expected_outputs: "expected/out"
```
I think this is all legal yaml, no?

Reply:
oh, yeah, I thought this might just be a little easier to parse and keep organized this way.

yeah, keeping everything as key-value mappings instead of using block sequences is cleaner, I agree.

Do you have a preference on reworking the format before/after the initial release?

Reply:
I would say it's unlikely I'll be able to get around to doing this refactoring before later next week, so if you prefer to get the tests updated here ASAP I'd be OK with merging this PR using the old format (with a v0.1 tag on Watt), and then update Watt to this convention for a v0.2 sometime in the near future.